### ISSUE TITLE
[feature] Add local sequence-logo drawer for meme seqs

### ISSUE THREAD

**Issue by Kiri2002** [biopython/biopython#5012]
according to [doc](http://biopython.org/DIST/docs/tutorial/Tutorial.html#sec283)
>If we have internet access, we can create a [weblogo](https://weblogo.berkeley.edu/):
 m.weblogo("mymotif.png")
We should get our logo saved as a PNG in the specified file.

biopython use web to draw logo. But now there are many good local drawer in python, for example:

[pyseqlogo](https://github.com/saketkc/pyseqlogo)
![image](https://github.com/biopython/biopython/assets/125180256/70d0ba9c-9c59-4169-81e6-fa285442eb30)

[Logomaker](https://github.com/jbkinney/logomaker),  it's [paper]( https://doi.org/10.1093/bioinformatics/btz921)
![image](https://github.com/biopython/biopython/assets/125180256/f3441979-e04e-457e-9489-6328c653de47)

so maybe we can pick some merge to biopython? Benefit a lot .

**Comment by mdehoon** [comment]
> so maybe we can pick some merge to biopython? Benefit a lot .

Yes I agree. But we have to be careful with the software license for those packages.
